src/main/java/com/trikzon/flora_doubling/Config.java

```diff
@@ -12,6 +12,8 @@
     public static class ConfigBean {
         public ArrayList<String> doublingFlora = new ArrayList<>();
         public boolean dispenser = true;
+        public boolean useSmallFlowersTag = true;
+        public boolean useTallFlowersTag = true;
```

Comment:
Have you tested this? Minecraft already has this behavior for tall flowers and I worry it will just do it twice.

If it does, add a check for when checking this tag that sees if it is a "minecraft:" block

Reply:
yup yup. I tested it only on small flowers originally because derp. But I just tested on tall flowers and it actually works and only produces one :)

Reply:
That's... unexpected.
Is it only running our code then?

Seems good then :)